packages/react-popover/e2e/Popover.e2e.ts

```diff
@@ -1,3 +1,5 @@
+export {};
```

Comment:
Looks strange, is it expected?

Reply:
Yeah since we build with `isolatedModules` I need an export/import to make this a module

Reply:
This is really smells 😯

Reply:
it's standard approach when using isolateModules/babel (especially in cypress).

we can turn it off for e2e projects. thoughts ?

Reply:
^ if agreed, it would be awesome if you could update it within migration generator 🙌

Reply:
@Hotell  PR #20759  should address this thread

Reply:
Applied those changes manually in [490af28](https://github.com/microsoft/fluentui/pull/20692/commits/490af28c9731bb52ba5d3d8ee85b44fb3458d41b)